db/postgres/sml_version.go

```diff
@@ -13,21 +13,12 @@ func CreateSMLVersion(ctx context.Context, smlVersion *SMLVersion) (*SMLVersion,
 
 	DBCtx(ctx).Create(&smlVersion)
 
-	for _, link := range smlVersion.Arch {
-		DBCtx(ctx).Create(&SMLArch{
-			ID:           util.GenerateUniqueID(),
-			SMLVersionID: smlVersion.ID,
-			Platform:     link.Platform,
-			Link:         link.Link,
-		})
-	}
```

Comment:
The only place that calls this function doesn't populate the `Targets` relation field, so that code was never used. The change is similar to how tags for mods are handled (set from the GQL resolver after the mod is created and saved).